smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/endpointsV2/ParameterGenerator.java

```diff
@@ -102,6 +107,12 @@ public String defaultAsCodeString() {
             case "boolean":
                 buffer += paramNode.expectBooleanMember("default").getValue() ? "true" : "false";
                 break;
+            case "StringArray":
```

Comment:
Minor, this case can be removed. This name in particular is not case sensitive, see [here](https://github.com/smithy-lang/smithy/blob/5da00e8e500175e75d35703e623961671c785d9d/smithy-rules-engine/src/main/java/software/amazon/smithy/rulesengine/language/syntax/parameters/ParameterType.java#L54)

Reply:
Updated in https://github.com/smithy-lang/smithy-typescript/pull/1376/commits/c62112d23e770d152af6b98f77e7f53c62865910